companies/exor.json

```diff
@@ -0,0 +1,19 @@
+{
+    "slug": "bitel",
+    "relevant-countries": [
+        "nl"
+    ],
+    "categories": [
+        "Holding company"
+    ],
+    "name": "Exor N.V.",
+    "address": "Gustav Mahlerplein 25, 1082 MS Amsterdam, The Netherlands",
+    "phone": "+49 521 514600",
+    "email": "media@exor.com",
+    "web": "https://www.exor.com/",
+    "sources": [
+        "https://www.exor.com/pages/contacts-1/"
+    ],
+    "suggested-transport-medium": "email",
```

Comment:
As we have no privacy-specific email, we cannot suggest that as the transport medium. You can just drop this line.

Reply:
i will resolve them